### ISSUE TITLE
Sussy baka looking website linked in repo

### ISSUE THREAD

**Issue by kxmpxtxnt** [Incendo/cloud#468]
Please remove this link

<img width="367" alt="image" src="https://github.com/Incendo/cloud/assets/78809981/a7fcb272-4f39-44d5-9e34-bee7780ecbc1">
<img width="876" alt="image" src="https://github.com/Incendo/cloud/assets/78809981/d3e188d3-377a-4138-8877-aac516eae663">

**Comment by lus** [comment]
🚨

**Comment by kxmpxtxnt** [comment]
Yes i know this is not really professional but having an invalid link to shady looking websites in a public well known repo is the real crime here

**Comment by cerus** [comment]
> Yes i know this is not really professional but having an invalid link to shady looking websites in a public well known repo is the real crime here

The real crime here is the title of this issue

**Comment by jpenilla** [comment]
#432
